backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3ApplicationConfiguration.java

```diff
@@ -234,6 +236,14 @@ public void enableGLDebugOutput (boolean enable, PrintStream debugOutputStream)
 		debugStream = debugOutputStream;
 	}
 
+	/** Configuration which will be set on if the {@link Os} is {@link Os/MacOsX}.
+	 *
+	 * @param useGlfwAsync Whether to use the experimental "glfw_async" library. This means you do not have to set the JVM argument
+	 *           "-XstartOnFirstThread". */
+	public void setMacOsX (boolean useGlfwAsync) {
```

Comment:
Am I understanding this right? It looks like:
```java
cfg.setMacOsX(true);
```
is exactly equivalent to:
```java
if (SharedLibraryLoader.os == Os.MacOsX) Configuration.GLFW_LIBRARY_NAME.set("glfw_async");
```
And setting it to false does nothing. The equivalent (longer) line doesn't need changes to libGDX to test, so hopefully we can get some macOS testers to try it out.